src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -266,7 +269,8 @@ public Config validate(Map<String, String> connectorConfigs) {
               result, Utils.SF_PRIVATE_KEY, " must be a valid PEM RSA private key");
           break;
         default:
-          throw e; // Shouldn't reach here, so crash.
+          Utils.updateConfigErrorMessage(result, Utils.SF_PRIVATE_KEY, "network is the because");
```

Comment:
> network is the because

Can you please change the error message?

Also, did we try to get the errorCode in case of timeouts? If there was a defined errorCode we should probably add a case in above switch and keep the `throw e;` as default case.

Reply:
this came by mistake, it was just for testing purpose.
reverting this change